embedded/servo_driver/servo_driver.hh

```diff
@@ -6,17 +6,28 @@
 //%deps(i2c)
 
 #include <stdint.h> // uint8_t etc
+#include <string>
 
 #include "embedded/pwm_driver/pwm_driver.hh"
 #include "third_party/i2c/i2c.h"
 
 
 class ServoDriver {
 public:
-	ServoDriver(char* dev);
+	ServoDriver(const int channel, PwmDriver &driver, const std::string &config_path);
 	int init();
-	void set_percentage(int channel, int percentage);
+	void set_percentage(int percentage);
+	int get_percentage() const;
+	void set_angle(float angle);
 
 private:
-        PwmDriver pwm_driver_;
+	int channel_;
+        PwmDriver &pwm_driver_;
```

Comment:
From talking over Slack, the way I'm using this is not an issue, but it could be used in a bad way.. it's important to ensure that pwm_driver_ *always* outlasts the lifetime of servo_driver